lib/fix_db_schema_conflicts/tasks/db.rake

```diff
@@ -1,19 +1,13 @@
 require 'shellwords'
 require_relative '../autocorrect_configuration'
 
-namespace :db do
-  namespace :schema do
-    task :dump do
-      puts "Dumping database schema with fix-db-schema-conflicts gem"
+Rake::Task['db:schema:dump'].enhance do
+  puts 'Dumping database schema with fix-db-schema-conflicts gem'
 
-      filename = ENV['SCHEMA'] || if defined? ActiveRecord::Tasks::DatabaseTasks
-        File.join(ActiveRecord::Tasks::DatabaseTasks.db_dir, 'schema.rb')
-      else
-        "#{Rails.root}/db/schema.rb"
-      end
-      autocorrect_config = FixDBSchemaConflicts::AutocorrectConfiguration.load
-      rubocop_yml = File.expand_path("../../../../#{autocorrect_config}", __FILE__)
-      `bundle exec rubocop --auto-correct --config #{rubocop_yml} #{filename.shellescape}`
-    end
-  end
+  # Copied from active_record/railties/databases.rake
+  filename = ENV['SCHEMA'] || File.join(ActiveRecord::Tasks::DatabaseTasks.db_dir, 'schema.rb')
```

Comment:
This may not work with older versions of Rails. I didn't put in the work to verify that this is the right move for Rails 4.2 or 5.0. We could add the `appraisal` gem if we want to be sure that we support older versions of Rails.